packages/ag-charts-enterprise/src/features/color-picker/colorPicker.ts

```diff
@@ -47,15 +42,9 @@ export class ColorPicker extends AnchoredPopover<ColorPickerOptions> {
         this.onChangeHide = options.onChangeHide;
 
         const { element, initialFocus } = this.createColorPicker(options);
-        const popover = this.showWithChildren([element], options);
+        const popover = this.showWithChildren([element], { initialFocus, ...options });
         popover.classList.add('ag-charts-color-picker');
         popover.setAttribute('role', 'dialog');
-
-        const { type, lastFocus } = this.ctx.focusIndicator.guessDevice(options.sourceEvent);
-        if (type === 'keyboard' && lastFocus !== undefined) {
-            initialFocus.focus();
-            this.lastFocus = lastFocus;
-        }
```

Comment:
These focus changes have been moved to the parent `Popover` class, but are only active if a `sourceEvent` and `initialFocus` options are passed in. The `Dialog` popover sets the `initialFocus` to the first tab if `Dialog.createTabs()` is called.